hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -383,9 +491,13 @@ async def last_known_status(self):
             return await self.status()  # updates _last_known_status
         return self._last_known_status
 
+    # TODO I think there's a decision here for what the semantics should be
+    # in a world of multiple concurrent batch updates.
+    # Should this be wait on the whole batch to reach completion or just
+    # the most recently-submitted update?
```

Comment:
Heh ya this is Danger if the job submits more jobs to its own batch. It would never complete

Reply:
I was just planning to type out this question. I don't think wait means anything sensible any more. Should we deprecate it and encourage waiting on a specific job?